zeppelin-plugins/launcher/k8s-standard/src/main/java/org/apache/zeppelin/interpreter/launcher/Kubectl.java

```diff
@@ -0,0 +1,157 @@
+/*
```

Comment:
if kubernetes java client from fabric can handle kubernetes yaml spec file the same way `kubectl` handles, why not? I tried to use official kubernetes java client first, and changed to use `kubectl` because it does not have `kubectl apply -f` equivalent functions.

Reply:
Usually with fabric8 API, applying of each resources has been tried by us. We will explore the usage of applying all resources. Initial reading of the code looks to be supporting it. However we will try and get back.

Reply:
@Leemoonsoo 
My colleague has put an example of the possible replacement under 
[zep-k8sctl](https://github.com/ajaygk95/zep-k8sctl)

Reply:
@nrchakradhar Cool! Thanks for sharing. 

One question, 

Let's say user is going to customize zeppelin-server.yaml, and interpreter.yaml. To mount volumns, add network restrictions, etc.

Therefore user might add additional sections in yaml, such as PersistentVolumeClaim, Secret, ConfigMap and so on. Also Kubernetes allows [CRD](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/), so there're some type of object we don't know at this point.

But [this code ](https://github.com/ajaygk95/zep-k8sctl/blob/master/zeppelin-spec-handler/src/main/java/com/example/k8s/fabric8/Kubectl.java#L107-L123) looks like need a modification if there's a new type of object to apply. Will there any any solution for this?

Reply:
The Issue with fabric8 library related to kubectl apply or delete of resources using a single API is working with [#1300](https://github.com/fabric8io/kubernetes-client/pull/1300) which is merged to master branch.
When it is available in the next release of fabric8io/kubernetes-client we can integrate the same. Using the above fix from fabric8 the change has been validated in the master branch of [zep-k8sctl](https://github.com/ajaygk95/zep-k8sctl).
Can you please let us know if we create new JIRA issues so that we follow up or we will follow some different procedure.

Reply:
Great news. Please feel free to create a JIRA issue and let me know if anything I can help.

Reply:
Okay. Sure